.github/workflows/tests-owners.yml

```diff
@@ -94,16 +108,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # empty = release that corresponds to current branch name
-        release: ['', 'eln']
```

Comment:
That `''` was there because our current required tests setup is expecting test with name `unit-tests`. However, this way we will have unit-tests (master) which is fine but will not work. 

I guess we should drop the current required tests configuration on the repository.

Reply:
Then we need to adapt the required tests, once the actions are in place. I think it's pretty easy to set up, with rules per branch.

Reply:
After discussion, let's keep that mechanism.

Reply:
To add more insights. It's not that simple to set specific test requirement for branches in GitHub. It's hard to match that for all future tests when the name of the test is changing, and creating new rules for each branching seems like a bad idea.